cpp/src/DataStorm/NodeI.cpp

```diff
@@ -154,11 +152,7 @@ NodeI::createSession(
     Ice::checkNotNull(subscriberSession, __FILE__, __LINE__, current);
 
     auto instance = _instance.lock();
-    if (!instance)
-    {
-        // Ignore the Node is being destroyed.
-        return;
-    }
+    assert(instance);
```

Comment:
Here we are dispatching a request, and the instance owns the OA and destroys it before shutdown, so we should always have an instance here.